lmdk/include/component.h

```diff
@@ -537,15 +537,18 @@ struct comp_driver_info {
  * Audio component base configuration from IPC at creation.
  */
 struct comp_ipc_config {
-	uint32_t core;		/**< core we run on */
-	uint32_t id;		/**< component id */
-	uint32_t pipeline_id;	/**< component pipeline id */
-	uint32_t proc_domain;   /**< processing domain - LL or DP */
+	uint32_t core;			/**< core we run on */
+	uint32_t id;			/**< component id */
+	uint32_t pipeline_id;		/**< component pipeline id */
+	uint32_t proc_domain;		/**< processing domain - LL or DP */
 	enum sof_comp_type type;	/**< component type */
-	uint32_t periods_sink;	/**< 0 means variable */
-	uint32_t periods_source;/**< 0 means variable */
-	uint32_t frame_fmt;	/**< SOF_IPC_FRAME_ */
-	uint32_t xrun_action;	/**< action we should take on XRUN */
+	uint32_t periods_sink;		/**< 0 means variable */
+	uint32_t periods_source;	/**< 0 means variable */
+	uint32_t frame_fmt;		/**< SOF_IPC_FRAME_ */
+	uint32_t xrun_action;		/**< action we should take on XRUN */
+#if CONFIG_IPC_MAJOR_4
+	uint32_t ipc_config_size;	/**< size of a config received by ipc */
+#endif
```

Comment:
please add more commit message for this change, there are multiple changes in this patch, and not sure it is copied from other source file or not.